single_group_playbooks/pre_deploy_checks.yml

```diff
@@ -66,6 +66,35 @@
       retries: 3
       delay: 10
       until: resolved_dependencies is not failed
+    - name: 'Find the path to ansible_builtin_runtime.yml'
+      ansible.builtin.find:
+        path: "{{ lookup('ansible.builtin.env', 'VIRTUAL_ENV') }}"
+        pattern: 'ansible_builtin_runtime.yml'
+        recurse: true
+      register: find_ansible_builtin_runtime_result
+      run_once: true  # noqa run-once
+      delegate_to: localhost
+      connection: local
+    - name: 'Report the found path to the ansible_builtin_runtime.yml config.'
+      ansible.builtin.debug:
+        msg: "INFO: Found {{ find_ansible_builtin_runtime_result.files | map(attribute='path') | first }}"
+      run_once: true  # noqa run-once
+      delegate_to: localhost
+      connection: local
+    - name: 'Patch ansible_builtin_runtime.yml for missing redirects.'
+      ansible.builtin.blockinfile:
+        dest: "{{ find_ansible_builtin_runtime_result.files | map(attribute='path') | first }}"
+        block: |2
```

Comment:
Apologies, as my block statements are really bad: but is this `2` here intentionally or by accident?

Reply:
That 2 is intentional and is required to make sure that block of code is inserted with the correct amount of indentation. It's a YAML thing and I had to look it up as well.